<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T3_AF_confirmExit</class>
 <widget class="QDialog" name="T3_AF_confirmExit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="_yesPushBtn_">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>_yesPushBtn_</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_noPushBtn_">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>_noPushBtn_</string>
   </property>
  </widget>
  <widget class="QLabel" name="_confirmLabel_">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>_confirmLabel_</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
